onnxruntime/core/providers/dml/DmlExecutionProvider/src/AbiCustomRegistry.cpp

```diff
@@ -544,7 +544,7 @@ HRESULT STDMETHODCALLTYPE AbiCustomRegistry::RegisterOperatorKernel(
 
         //
         // For backward compatibility, this does not propagate errors for external operators
-        m_kernelRegistry->RegisterCustomKernel(create_info);
+        static_cast<void>(m_kernelRegistry->RegisterCustomKernel(create_info));  // ignore result
```

Comment:
not sure if ignoring the Status is the right thing to do here, but that's how I interpreted the comment above this line.
can someone familiar with this code please confirm?